src/Pagination/Pagination.scss

```diff
@@ -71,6 +85,11 @@
     transition: none;
     line-height: $pagination-line-height;
   }
+
+  &.active .page-link:focus {
+    background-color: $pagination-focus-color !important;
+    color: $pagination-bg !important;
```

Comment:
[curious] Anyway to avoid using `!important` here, maybe getting the selector to be more specific in some way?

Reply:
Done